EmbeddedScripts.CSharp/Common/ResolvingCodeGenerator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Linq;
+using EmbeddedScripts.Shared;
+
+namespace EmbeddedScripts.CSharp.Common
+{
+    public static class ResolvingCodeGenerator
+    {
+        public static string GenerateUsingLines(Container container) =>
+            GenerateCode(container, (_, type) => 
+                $"using {type.Namespace};");
+
+        public static string GenerateResolveLines(Container container) =>
+            GenerateCode(container, (alias, type) =>
+                $"var {alias} = Container.Resolve<{type.Name}>(\"{alias}\");");
```

Comment:
Do you really need `using` stmts injections? You can probably just use full qualified type in resolve container and `var` will let compiler to calculate the type. For example, 
var file = Container.Resolve<System.IO.File>